pages/sign-up.js

```diff
@@ -29,26 +48,13 @@ const SignUp = () => {
                         </p>
                       </div>
                       <div className="">
-                        <form action="#" method="POST" className="space-y-1">
-                          <div>
-                            <Input label="Email address" name="email" type="email" className="w-full" />
-                          </div>
-                          <div className="">
-                            <Input
-                              label="Password"
-                              name="password"
-                              type="password"
-                              Icon={<PasswordCheckIcon width="18px" height="18px" fill="#292D32" />}
-                            />
-                          </div>
-                          <div className="">
-                            <Input
-                              label="Confirm Password"
-                              name="confirm-password"
-                              type="password"
-                              Icon={<PasswordCheckIcon width="18px" height="18px" fill="#292D32" />}
-                            />
-                          </div>
+                        <HookForm onSubmit={onSubmit} schema={SignUpSchema} className="space-y-1">
```

Comment:
`className` is used here but it is not used in `HookForm`. Can we remove the className or use it in `HookForm`. Any option you go with works for me.